packages/@react-native-windows/tester/src/js/examples-win/Button/ButtonExample.windows.js

```diff
@@ -219,6 +219,8 @@ exports.examples = [
                 accessibilityLabel="Press to submit your application!"
                 accessibilityRole="Button"
                 accessibilityHint="Submit Button"
+                accessibilityPosInSet={1}
```

Comment:
Should we also add these to: \vnext\src-win\Libraries\Components\...?

Reply:
No that won't be needed because these are just the example files, not the actual source! The examples live in packages/@rnw and the source for the button control lives in vnext/src-win...